### PR TITLE
DDT-1130: draft mielsremaning mouseover

### PR DESCRIPTION
**Pull request by Abdiiir**
added vesselDraft and milesRemainingToDestination to the UI as mouse-over tooltip in both the Vessel Name and the IMO Number fields in the transportcalls.

Depends on #165

### CODE REVIEW

File: src/app/controller/pipes/vessel-tooltip.pipe.ts [hunk 1]
```diff
@@ -8,15 +7,20 @@ import { Vessel } from "../../model/portCall/vessel";
 })
 export class VesselTooltipPipe implements PipeTransform {
 
-  transform(vessel: Vessel): string {
+  transform(vessel: Vessel, milesRemainingToDestination: number): string {
```
Comment (HenrikHL) [inline src/app/controller/pipes/vessel-tooltip.pipe.ts:10]: I have called it `milesToDestinationPort` in the API specs

File: src/app/controller/pipes/vessel-tooltip.pipe.ts [hunk 2]
```diff
@@ -8,15 +7,20 @@ import { Vessel } from "../../model/portCall/vessel";
...
+    let tooltip = "";
     if (vessel) {
-      var tooltip = "";
       if (vessel.type) tooltip += `Type: ${vessel.type}\n`;
       if (vessel.width) tooltip += `Width: ${vessel.width}\n`;
       if (vessel.length) tooltip += `Length: ${vessel.length}\n`;
       if (vessel.vesselCallSignNumber) tooltip += `Call sign: ${vessel.vesselCallSignNumber}\n`;
-      return tooltip
+      if (milesRemainingToDestination) tooltip += `milesRemainingToDestination: ${milesRemainingToDestination}\n`;
+      if (vessel.vesselDraft) tooltip += `Draft: ${vessel.length}\n`;
```
Comment (HenrikHL) [inline src/app/controller/pipes/vessel-tooltip.pipe.ts:19]: I would assume we need a unit of measuer here (Feet or Meters) - the value can be read in the `vesselDraftUnit` attribute (might not be implemented yet in backend)

File: src/app/model/jit/transport-call.ts [hunk 3]
```diff
@@ -29,4 +29,5 @@ export interface TransportCall {
   atdBerthDateTime?: Date;
   latestEventCreatedDateTime?: Date;
   transportCallID: string;
+  milesRemainingToDestination?: number;
```
Comment (HenrikHL) [inline src/app/model/jit/transport-call.ts:32]: To be renamd to `milesToDestinationPort`

File: src/app/controller/pipes/vessel-tooltip.pipe.ts (hunk 1 above)
Comment (Abdiiir) [reply src/app/controller/pipes/vessel-tooltip.pipe.ts:10]: This is in extension of discussion I brought up yesterday about alignment of fields naming. `milesRemainingToDestination `is the given field name in the backend.

File: src/app/controller/pipes/vessel-tooltip.pipe.ts (hunk 2 above)
Comment (Abdiiir) [reply src/app/controller/pipes/vessel-tooltip.pipe.ts:19]: same as above

File: src/app/controller/pipes/vessel-tooltip.pipe.ts [hunk 4]
```diff
@@ -8,15 +7,20 @@ import { Vessel } from "../../model/portCall/vessel";
...
 export class VesselTooltipPipe implements PipeTransform {
 
-  transform(vessel: Vessel): string {
+  transform(vessel: Vessel, milesRemainingToDestination: number): string {
+
+    let tooltip = "";
     if (vessel) {
-      var tooltip = "";
       if (vessel.type) tooltip += `Type: ${vessel.type}\n`;
       if (vessel.width) tooltip += `Width: ${vessel.width}\n`;
```
Comment (HenrikHL) [inline src/app/controller/pipes/vessel-tooltip.pipe.ts:15]: Same comment as below for vesselDraft - a unit of measure is needed

File: src/app/model/jit/transport-call.ts (hunk 3 above)
Comment (Abdiiir) [reply src/app/model/jit/transport-call.ts:32]: same as above

**Review by HenrikHL — APPROVED** [review]

**Pull request merged**